cuenca/http/client.py

```diff
@@ -41,31 +43,20 @@ def __init__(self):
         api_secret = os.getenv('CUENCA_API_SECRET', '')
         self.basic_auth = (api_key, api_secret)
 
-        # IAM auth
-        aws_access_key = os.getenv('AWS_ACCESS_KEY_ID', '')
-        aws_secret_access_key = os.getenv('AWS_SECRET_ACCESS_KEY', '')
-        aws_region = os.getenv('AWS_DEFAULT_REGION', AWS_DEFAULT_REGION)
-        if aws_access_key and aws_secret_access_key:
-            self.iam_auth = AWSRequestsAuth(
-                aws_access_key=aws_access_key,
-                aws_secret_access_key=aws_secret_access_key,
-                aws_host=self.host,
-                aws_region=aws_region,
-                aws_service=AWS_SERVICE,
-            )
-
```

Comment:
is there a way to configure the jwt auth using env vars vs manually calling `.configure()`?

Reply:
Tokens are generated using basic auth credentials and only last for 7 days. If we configure the token using env vars, we should have to change it every 7 days manually. Using `configure()` the token is generated automatically every time it's about to expire.